src/main/java/br/com/caelum/viagens/administrativo/model/Companhia.java

```diff
@@ -0,0 +1,59 @@
+package br.com.caelum.viagens.administrativo.model;
+
+import java.time.Instant;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.validation.constraints.NotBlank;
+
+import org.springframework.format.annotation.DateTimeFormat;
+
+import com.sun.istack.NotNull;
+
+@Entity
+public class Companhia {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+
+	@NotBlank
+	private String nome;
+	
+	@DateTimeFormat(pattern="dd/MM/yyyy HH:mm:ss")
```

Comment:
*Idealmente, todo código escrito deveria ser chamado por alguém. Se não tem ninguém chamando, ele não deveria existir.*

Lembra que essa anotação só precisa ser usada esse atributo for ser preenchido ou exibido diretamente pelo 
Spring, o que não é o caso. Dentro do seu model, a não precisa de formato :).

Reply:
Mesmo o NotNull é aqui tende ao exagero, apesar que eu também usaria huahaua.

Reply:
Sim, erro meu. Eu coloquei porque achei que, por um momento, eu estava esquecendo de colocar algo quando a @JsonFormat do jackson não funcionou no meu DTO de output por causa do DayOfMonth - que aliás, eu apaguei porque não consegui fazer funcionar. Ainda não entendi se esqueci de algo ou eu precisaria fazer um converter extra (vou pesquisar depois).

Sobre a @NotNull é verdade! Acho que acabei colocando antes de iniciar a variável, e acabei deixando, hehe.